docs/README.adoc

```diff
@@ -0,0 +1,99 @@
+= ShiftStack CI
+
+Each OpenShift on OpenStack continuous integration test job runs on one of two clouds:
+
+* MassOpenCloud: OSP 13
+* Vexxhost (private cloud offering): OpenStack based on 16
+
+Currently, we have three different test suites:
+
+* *Parallel*: The Conformance test suite, executed parallelizing tests
+* *Serial*: The Conformance test suite, executed one job at a time
+* *Early*: Smoke tests
+
+Currently, our jobs all target the same infrastructure:
```

Comment:
Not sure I understand what `same infrastructure` means in that context. We can tweak the flavors, or use different parameters for each of our jobs.
I'd rephrase by saying  `Currently, our jobs deploy on clouds with equivalent characteristics`